handlers/network_handler.py

```diff
@@ -0,0 +1,20 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
+
+
+def handle(event, context):
+    response = None
+    if event['pathParameters'] is not None:
+        response = redis_client.get(event['pathParameters']['hash']).decode()
+    else:
+        response = json.dumps([k.decode() for k in redis_client.keys()])
```

Comment:
To avoid format errors (unicode/bytes, json, etc.) you could use something like msgpack to pack the python object as bytes, and rebuild the original object easily when retrieving.

Reply:
Created an issue for it: https://github.com/HathorNetwork/hathor-explorer-service/issues/10

I like the idea, i think we must use it. But we don't need right now.